tsl/src/chunk_api.c

```diff
@@ -1133,9 +1136,13 @@ static void
 stats_process_context_finish(StatsProcessContext *ctx)
 {
 	hash_destroy(ctx->htab);
+	MemoryContextDelete(ctx->per_tuple_mcxt);
 }
 
-static void
+void chunk_process_remote_colstats_row(StatsProcessContext *ctx, TupleFactory *tf,
+									   TupleDesc tupdesc, PGresult *res, int row,
```

Comment:
inline declaration of a the function signature? By mistake, I presume?

Reply:
yes, removed during testing, and forgot to revert.